sdcm/nemesis.py

```diff
@@ -1186,8 +1186,8 @@ def _terminate_cluster_node(self, node):
         self.cluster.terminate_node(node)
         self.monitoring_set.reconfigure_scylla_monitoring()
 
-    def disrupt_nodetool_decommission(self, add_node=True, disruption_name=None):
```

Comment:
cannot remove `disruption_name` parameter

Reply:
why? it was used only from the seed node decommission